test/scholar/linear/logistic_regression_test.exs

```diff
@@ -5,34 +5,40 @@ defmodule Scholar.Linear.LogisticRegressionTest do
 
   test "Pima Indians Diabetes Data - binary logistic regression test" do
     {x_train, x_test, y_train, y_test} = Datasets.get(:pima)
+    y_train = Nx.squeeze(y_train, axes: [1])
+    y_test = Nx.squeeze(y_test, axes: [1])
 
     model =
       LogisticRegression.fit(x_train, y_train,
         num_classes: 2,
-        iterations: 100,
-        learning_rate: 0.0085
+        iterations: 14,
+        learning_rate: 0.01
```

Comment:
I don't know why, but when I set here the number of iterations bigger than 14, then there is 0 division in grad.

Reply:
Ok, the problem was with `Nx.log` in cross entropy. Some probabilities were really close to 0, which in grad produce 0 division. When I change to Axon `log_softmax` it works correct.